### PR TITLE
SDIT-1174 prison-api CaseNoteResource

### PR DESCRIPTION
**Pull request by steverendell**
Switch to system token

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,70 +103,14 @@ docker run -p 3000:3000 -d \
...
-
-  - Ensure that your chrome and chromedriver versions match, and are set in the build.gradle file.
-
-  - Open a Spock Specification (`uk.gov.justice.digital.hmpps.keyworker.specs.LoginSpecification` for example). 
-
-  - The IDE gutter should now display the green 'run' icons for both the class and each of its test methods.
-
-  - Click the green run icon to start test
-
-## Cypress integration tests
```
Comment (steverendell) [inline README.md:162]: Removing sections referring to the old Geb / groovy tests

File: docker-compose-test.yaml [hunk 2]
```diff
@@ -2,12 +2,19 @@ version: '3.1'
...
   
   redis:
     image: redis
+    networks:
+      - hmpps
     ports:
       - 6379:6379
+
+networks:
+  hmpps:
```
Comment (steverendell) [inline docker-compose-test.yaml:20]: I found i needed the network config in my local env

File: gradle/wrapper/gradle-wrapper.jar
Comment (mikehalmamoj) [inline gradle/wrapper/gradle-wrapper.jar:1]: Thought you might have upgraded this rather than deleting?

**Review by mikehalmamoj — APPROVED** [review]

File: gradle/wrapper/gradle-wrapper.jar
Comment (steverendell) [reply gradle/wrapper/gradle-wrapper.jar:1]: Gradle was only needed for the old groovy tests so it can go now it uses cypress :)

**Pull request merged**